tests/test_arg_parse.py

```diff
@@ -19,6 +19,7 @@ def test_everything():
         verbose=True,
         version=False,
         include_prereleases=False,
+        dry_run=False,
```

Comment:
Why this?

Reply:
Tests fail otherwise. 
```
AssertionError: assert (Namespace(al...on=False), []) == (Namespace(alg...on=False), [])
E         At index 0 diff: Namespace(algorithm='sha512', dry_run=False, include_prereleases=False, packages=['example', 'another-example'], python_version=['3.5'], requirements_file='reqs.txt', verbose=True, version=False) != Namespace(algorithm='sha512', include_prereleases=False, packages=['example', 'another-example'], python_version=['3.5'], requirements_file='reqs.txt', verbose=True, version=False)
```

Reply:
I guess it's there to test with all options set.